app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/TestRunner.ts

```
@@ -86,16 +89,20 @@ export class TestRunner {
     }
     this._wasRenderedNull = !component;
     this._renderLock.lock = true;
-    this._renderHook(component);
+    try {
+      this._renderHook(component);
+    } catch (e) {
+      console.log(e);
```

Do we ignore all exceptions? 🤔